actors/miner/tests/apply_rewards.rs

```diff
@@ -34,14 +33,15 @@ fn funds_are_locked() {
     let rwd = TokenAmount::from_atto(1_000_000);
     h.apply_rewards(&rt, rwd, TokenAmount::zero());
 
-    let expected = TokenAmount::from_atto(750_000);
+    let expected = TokenAmount::from_atto(750_000).add(rt.policy().new_miner_deposit.clone());
     assert_eq!(expected, h.get_locked_funds(&rt));
 }
 
 #[test]
 fn funds_vest() {
     let h = ActorHarness::new(PERIOD_OFFSET);
-    let rt = h.new_runtime();
+    let mut rt = h.new_runtime();
+    rt.policy.new_miner_deposit = TokenAmount::zero();
```

Comment:
I think it's a problem that this test is touched at all, and that an unrealistic value is used for the deposit. Can you explain why (in code comments) ?